File: src/main/java/com/example/demo/xxxx/service/UserService.java
package com.example.demo.xxxx.service;

import com.example.demo.xxxx.bean.OrderUserBean;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public abstract class UserService implements JpaRepository<OrderUserBean, Integer> {
    public abstract int signUp(OrderUserBean orderUser);

    public abstract OrderUserBean signIn(String account , String password);

    public abstract List<OrderUserBean> findAllBean();

    public abstract OrderUserBean findByAccount(String account);
}

File: src/main/java/com/example/demo/xxxx/service/UserService.kt
package com.example.demo.xxxx.service

import com.example.demo.xxxx.bean.UserOrder


interface UserService {


    fun signUp(user: UserOrder): Int

    fun signIn(account: String, password: String): UserOrder?

    fun findAllBean(): List<UserOrder>?

    fun findByAccount(account: String): UserOrder

}
